lib/scrum/priority_name.rb

```diff
@@ -0,0 +1,15 @@
+module Scrum
+  class PriorityName
```

Comment:
This class has no initializing code, no instance variable and no state. Only two helper methods. Maybe it is better to just make them class methods. This way we won't have to initialize this class before using it (for no reason).